servers/rendering/renderer_rd/cluster_builder_rd.h

```diff
@@ -235,7 +235,7 @@ class ClusterBuilderRD {
 		Transform xform = view_xform * p_transform;
 
 		float radius = xform.basis.get_uniform_scale();
-		if (radius > 0.98 || radius < 1.02) {
+		if (radius > 0.98 && radius < 1.02) {
```

Comment:
With this one, I'm not sure.
Should be `(radius > 0.98 && radius < 1.02)` or maybe `(radius < 0.98 || radius > 1.02)`

Reply:
We will need to check with @reduz